src/java.desktop/macosx/classes/com/apple/laf/AquaLookAndFeel.java

```diff
@@ -869,7 +870,7 @@ public Object createValue(UIDefaults defaultsTable) {
             "TabbedPane.tabsOverlapBorder", Boolean.TRUE,
             "TabbedPane.selectedTabTitlePressedColor", selectedTabTitlePressedColor,
             "TabbedPane.selectedTabTitleDisabledColor", selectedTabTitleDisabledColor,
-            "TabbedPane.selectedTabTitleNormalColor", selectedTabTitleNormalColor,
+            "TabbedPane.selectedTabTitleNormalColor", System.getProperty("os.version").contains("10.16") ? selectedControlTextColor : selectedTabTitleNormalColor,
```

Comment:
That doesn't look very robust. What will happen on the next version of macOS ?

Reply:
Check for BigSur and above is added